docs/disable-parameterized-tests.adoc

```diff
@@ -0,0 +1,250 @@
+		"Then let them toll the passing-bell,;Then of your servitude be free,",
+		"The clock may stop, its hands fall still,;And time be over then for me!"
+}, delimiter = ';')
+void disableAllContains(String line, String line2) {
+}
+----
+
+The test `disableAllContains` ordinarily would run four times, but the second execution gets disabled because both arguments contain "the" (the second argument as part of "there").
+Using the same test with a different annotation would look like this:
+
+[source,java]
+----
+@DisableIfAnyParameter(contains = "Then")
+@ParameterizedTest
+@CsvSource(value = {
+		"If the swift moment I entreat:;Tarry a while! You are so fair!",
+		"Then forge the shackles to my feet,;Then I will gladly perish there!",
+		"Then let them toll the passing-bell,;Then of your servitude be free,",
+		"The clock may stop, its hands fall still,;And time be over then for me!"
+}, delimiter = ';')
+void disableAnyContains(String line, String line2) {
+}
+----
+
+The test `disableAnyContains` ordinarily would run four times, but the second and third executions get disabled because an argument contains "Then".
+The last execution does not get disabled, because the extension is case-sensitive.
+
+You can specify more than one substring at a time:
+
+[source, java]
+----
+@DisableIfAnyParameter(contains = { "Then", "then" })
+@ParameterizedTest
+@CsvSource(value = {
+		"If the swift moment I entreat:;Tarry a while! You are so fair!",
+		"Then forge the shackles to my feet,;Then I will gladly perish there!",
+		"Then let them toll the passing-bell,;Then of your servitude be free,",
+		"The clock may stop, its hands fall still,;And time be over then for me!"
+}, delimiter = ';')
+void disableAnyContains(String line, String line2) {
+}
+----
+
+The extension disables the second, third and fourth executions because an argument contains either "Then" or "then".
+
+`@DisableIfParameter` requires you to target a specific parameter.
+You can do this in three ways:
+
+- By a `name` https://docs.oracle.com/javase/8/docs/api/java/lang/reflect/Parameter.html#isNamePresent--[if parameter naming information is present].
+- By an explicit `index`, starting from 0.
+- By an implicit index.
+
+Using both `name` and `index` in a single `@DisableIfParameter` annotation is not permitted.
+
+==== Targeting by `name`
+
+If naming information is included during compilation, you can target parameters by their name.
+
+[source, java]
+----
+@DisableIfParameter(name = "line2", contains = "swift")
+@ParameterizedTest
+@CsvSource({
+		"If the swift moment I entreat:;Tarry a while! You are so fair!",
+		"Then forge the shackles to my feet,;Then I will gladly perish there!"
+})
+void targetName(String line, String line2) {
+}
+----
+
+The test gets executed two times because we explicitly targeted the second parameter, which never contains the word "swift".
+
+==== Targeting by `index`
+
+You can target your parameters with their index, starting from 0.
```

Comment:
I personally like to write this `.. starting from 0 (zero)` to show that I don't have a typo.